src/System.Globalization/tests/CharUnicodeInfo/CharUnicodeInfoTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class CharUnicodeInfoGetUnicodeCategoryTests
+    {
+        [ActiveIssue(7351, PlatformID.AnyUnix)]
+        [Fact]
+        public void GetUnicodeCategory()
```

Comment:
I think we shouldn't disable the whole test in Linux because of these 2 code points. we may just exclude these 2 code points and add small test case to test these 2 codepoints and disable it on Linux